### PR TITLE
Fix Bug with Alias Account

### PR DESCRIPTION
**Pull request by ThreeAndTwo**
1. Bug Fix: Alias Account Overwriting
2. Code Improvement: Use of Constants

### CODE REVIEW

**Comment by RonTuretzky** [comment]
@uri-lightblocks thought?

File: internal/operatorcli/operator_key.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	aliasKeystore          string
+	aliasECDSAPassphrase   string
+}
+
+type priKeys struct {
+	operatorECDSA *ecdsa.PrivateKey
+	aliasECDSA    *ecdsa.PrivateKey
+}
+
+func NewMangeKey(
```
Comment (uri-lightblocks) [inline internal/operatorcli/operator_key.go:26]: passPhrase should be the same for all keys

Comment (uri-lightblocks) [inline internal/operatorcli/operator_key.go:26]: rename function name (and other references from mange to manage - add `a`)

File: internal/operatorcli/operator_key.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+type mangeKey struct {
+	keyStorePath           string
+	operatorKeystore       string
+	operatorECDSPassphrase string
+	aliasECDSA             string
+	aliasKeystore          string
+	aliasECDSAPassphrase   string
+}
+
+type priKeys struct {
```
Comment (uri-lightblocks) [inline internal/operatorcli/operator_key.go:21]: rename to privateKeys

Comment (uri-lightblocks) [inline internal/operatorcli/operator_key.go:21]: add bls private key

File: internal/operatorcli/operator_key.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+		keyStorePath:           keyStorePath,
+		operatorKeystore:       operatorECDSAFile,
+		operatorECDSPassphrase: operatorECDSPassphrase,
+		aliasECDSA:             aliasECDSA,
+		aliasKeystore:          aliasECDSAFile,
+		aliasECDSAPassphrase:   aliasECDSAPassphrase,
+	}
+}
+
+func (m *mangeKey) getPrivateKeys() (priKeys, error) {
```
Comment (uri-lightblocks) [inline internal/operatorcli/operator_key.go:44]: add reading the bls private key

File: internal/operatorcli/operator_key.go [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+type priKeys struct {
+	operatorECDSA *ecdsa.PrivateKey
+	aliasECDSA    *ecdsa.PrivateKey
+}
+
+func NewMangeKey(
+	keyStorePath string,
+	operatorECDSAFile string,
+	operatorECDSPassphrase string,
```
Comment (uri-lightblocks) [inline internal/operatorcli/operator_key.go:29]: missing `A` after `ECDS`

File: internal/operatorcli/operator_key.go [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+	return operatorECDSAPair, nil
+}
+
+func (m *mangeKey) getAliasPrivateKey() (*ecdsa.PrivateKey, error) {
+	aliasPrivateKey, err := eigensdkecdsa.ReadKey(filepath.Join(m.keyStorePath, m.aliasKeystore), m.aliasECDSAPassphrase)
+	if err != nil {
+		var ecdsaPair *ecdsa.PrivateKey
+		// generate
+		if m.aliasECDSA != "" {
+			if m.aliasECDSA[:2] == "0x" || m.aliasECDSA[:2] == "0X" {
```
Comment (uri-lightblocks) [inline internal/operatorcli/operator_key.go:78]: check the length of aliasECDSA

**Comment by ThreeAndTwo** [comment]
The code has been updated according to the review comments, and a re-review is requested.

File: internal/operatorcli/operator_cli.go [hunk 6]
```diff
@@ -478,30 +478,14 @@ func RunEOChainSetAlias(c *cli.Context) error {
...
-		ecdsaPair, err = crypto.GenerateKey()
-		if err != nil {
-			return cli.Exit(fmt.Sprintf("Failed to generate ECDSA key %v", err), 1)
-		}
-	}
 
-	// Save the private key to a file
-	if err = eigensdkecdsa.WriteKey(filepath.Join(keyStorePath, "ecdsaAliasedEncryptedWallet.json"), ecdsaPair, passphrase); err != nil {
-		return cli.Exit(fmt.Sprintf("Error writing the ecdsaAliasedEncryptedWallet.json file %v", err), 1)
+	_mangeKey := NewMangeKey(keyStorePath, flag.EncryptedOperatorECDSAFile, passphrase,
```
Comment (uri-lightblocks) [inline internal/operatorcli/operator_cli.go:482]: I didn't see where you encrypt the alias private key

**Comment by RonTuretzky** [comment]
This has been resolved upstream by the separation of alias generation and declaration.

**Pull request closed without merging**